src/test/ui/const-generics/condition-in-trait-const-arg.rs

```diff
@@ -1,7 +1,10 @@
+// Checks whether conditions in traits can be evaluated.
```

Comment:
This doesn't seem right to me :thinking: This only checks that `impl Trait<{anon_const}>` evaluates successfully.